src/server/plugins/engine/components/DatePartsField.test.ts

```diff
@@ -73,11 +79,33 @@ describe('Date parts field', () => {
         }
       ]
     }
-    const underTest = new DatePartsField(def)
+
+    const underTest = new DatePartsField(def, {} as FormModel)
     const returned = underTest.getViewModel({}, errors)
     expect(returned.errorMessage?.text).toBe('Day must be a number')
     expect(underTest.getViewModel({}).errorMessage).toBeUndefined()
   })
+  test('Condition evaluation used yyyy-MM-dd format', () => {
+    const datePartsFieldComponent = {
+      title: 'Example checkboxes',
+      name: 'myComponent',
+      type: ComponentType.DatePartsField,
+      options: {},
+      schema: {}
+    } satisfies DatePartsFieldFieldComponent
```

Comment:
Did linting fail with the general purpose `satisfies ComponentDef` here?

Reply:
Nah, I just referenced another test file and didn't notice the ComponentDef earlier in this one. I've switched to ComponentDef for consistency within the file.

Reply:
Yeah, either works for me, can stick to one only most of the time. Thanks